lib/broadway/producer.ex

```diff
@@ -71,7 +70,7 @@ defmodule Broadway.Producer do
 
   The goal of this callback is to manipulate the general topology options,
   if necessary at all, and introduce any new child specs that will be
-  started **before** the producers supervisor in Broadway's supervision tree.
+  started **before** the producer's supervisor in Broadway's supervision tree.
```

Comment:
This supervisor supervises multiple producers not just one.


```suggestion
  started **before** the producers supervisor in Broadway's supervision tree.
```

Reply:
I changed it to plural possessive if that's ok.